.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy Storybook
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-storybook:
+    runs-on: ubuntu-latest
+    name: Deploy Storybook
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
```

Comment:
Just noting this is another place we should adjust for any node version changes (#72)